source/system-administrators/activities/security/security-change-defaults.rst

```diff
@@ -39,7 +39,7 @@ Crafter CMS installations are pre-configured with default passwords, tokens, key
 
      |
 
-* Replace default values for the DB root password and the DB ``crafter`` user password
+* Replace default values for the DB root password and the DB ``crafter`` user password before starting Crafter CMS.  Remember that these values should **not** be changed after the DB has been installed
```

Comment:
What happens if they want to change after they start (for example the AWS AMI product line)?
Perhaps we can explain that if you want to change after initial start, you must do XYZ.